libretro/LibretroGraphicsContext.cpp

```diff
@@ -80,35 +80,44 @@ void LibretroGraphicsContext::LostBackbuffer() { draw_->HandleEvent(Draw::Event:
 
 LibretroGraphicsContext *LibretroGraphicsContext::CreateGraphicsContext() {
 	LibretroGraphicsContext *ctx;
-
-	ctx = new LibretroGLContext();
-
-	if (ctx->Init()) {
-		return ctx;
+	
+	unsigned preferred; // This will be set to a const value if GET_PREFERRED_HW_RENDER is not supported
+	if (!Libretro::environ_cb(RETRO_ENVIRONMENT_GET_PREFERRED_HW_RENDER, &preferred)) preferred = 0xFFFFFFFF;
+	
+	if (preferred == RETRO_HW_CONTEXT_OPENGL || preferred == RETRO_HW_CONTEXT_OPENGL_CORE || preferred == 0xFFFFFFFF) {
+		ctx = new LibretroGLContext();
```

Comment:
If it's `RETRO_HW_CONTEXT_OPENGL_CORE`, please call `SetGLCoreContext(true);` from `gfx_es2/gpu_features.h`.  This will cause us to avoid `GL_EXTENSIONS` and use VAOs, etc.  It'd also be best to set `glewExperimental = true;` before calling `glewInit()`.

We actually default to a core context on Windows for standalone PPSSPP, but only if we can obtain a core context from Windows.  It's generally faster on NVIDIA, and I think it's been well-received elsewhere too.

-[Unknown]

Reply:
Where should i call those? The PR was supposed to just mimic old code.